### ISSUE TITLE
G2D Data Source Analysis

### ISSUE THREAD

**Issue by sagehrke** [monarch-initiative/monarch-app#707]
A detailed Gene to Disease Monarch Ingest data source analysis is needed. @madanucd please enter the information here that you are working on.

**Comment by madanucd** [comment]
![image](https://github.com/monarch-initiative/monarch-ingest/assets/142266502/ebf5f075-5381-4cc6-9a13-31b8685b160b)

**Comment by madanucd** [comment]
For the latest developments in G2D analysis, please refer to the following link: [G2D Analysis Update](https://docs.google.com/presentation/d/1QP66I1ld6HsZLWFiSb2ERsNOYUfjgpj8/edit#slide=id.p1)

Approximately 1400 G2D associations from GENCC have been excluded from Monarch/HPOA. These exclusions stem from diverse sources that have not been accounted for within the HPOA or Monarch databases.

**Comment by madanucd** [comment]
Please find the latest updates regarding G2D data source analysis:
•	The notebook dedicated to G2D data source analysis, which parses 5 data sources and generates visualizations, is available at the following link: [Notebook Link](https://colab.research.google.com/drive/1LHRCK_H9-mTHCh5d1FefqWQWLzKy-wgd?usp=sharing).
•	The [slides](https://docs.google.com/presentation/d/1QP66I1ld6HsZLWFiSb2ERsNOYUfjgpj8/edit#slide=id.g26d8847266a_0_80) from previous presentation have been recently updated to include an UpSet Plot, providing a comprehensive view of the data.
Additionally, the list of excluded G2D associations from GENCC can be accessed through the following link: [Excluded Associations from GENCC](https://drive.google.com/file/d/1RpjWT3zFEdYIfYkZtjB8RKYI0Sd3WnDD/view?usp=sharing).

**Comment by sagehrke** [comment]
@julesjacobsen @cmungall @kevinschaper @monicacecilia FYI 👀 ⬆️

**Comment by kevinschaper** [comment]
I'm hoping we can get to a consensus/context for what to do next with our GENCC analysis. Given that we want one source of truth for G2D within Monarch, I'm curious if we're considering eventually bringing GENCC into the HPOA pipeline?  

cc:@iimpulse

**Comment by iimpulse** [comment]
Lets figure out what the right approach is. It will be trivial to bring some/all of these guys in. Can someone set up a meeting to review gencc and the level of filtering we want to use.

**Comment by julesjacobsen** [comment]
One likely blocker issue is that the GenCC file appears to be created from submissions from the data sources (e.g. ClinGen, PanelApp) and these appear to be unscheduled and infrequent such that the data in the GenCC file is out of date when compared to the underlying data sources. Consequently, it would probably make more sense to take these annotations from the original data sources directly. Some of these aren't available other than through the GenCC, so we'll end up to a choice of ClinGen, the PanelApps, G2P or whatever.

**Comment by kevinschaper** [comment]
@julesjacobsen Would it make sense to just restrict a gencc ingest to the subset that we can only get from them, and live with the synchronization being a little behind?

**Comment by madanucd** [comment]
I agree with @julesjacobsen  that the timing of releases is crucial for these sources. Although GENCC provides submitted data for each association, it might be from older releases.

GENCC classifies sources into the following categories:

- Organizations that provide online resources (e.g., ClinGen, DECIPHER, Genomics England PanelApp, OMIM, Orphanet, PanelApp Australia, TGMI’s G2P)
- Diagnostic laboratories that share internal curated gene-level knowledge (e.g., Ambry, Illumina, Invitae, Myriad Women’s Health, Mass General Brigham Laboratory for Molecular Medicine)

As suggested by Jules, we may consider excluding data from diagnostic labs.

**All gene-to-disease (G2D) source analyses can now be found in our GitHub `source-data-analysis` repository:**

- [Documentation](https://monarch-initiative.github.io/source-data-analysis/g2d/): Detailed descriptions of methods, data sources, and normalization techniques.
- [Scripts](https://github.com/monarch-initiative/source-data-analysis): Python scripts for the complete analysis workflow.
- [Releases](https://github.com/monarch-initiative/source-data-analysis/releases): Updated releases including visualizations, consolidated data, and mappings.

Currently, @iimpulse  is addressing concerns about Mendelian conditions with multiple gene associations. I will close this ticket once we receive his clarification. 

For the latest results and updates on G2D analysis, please refer to the [GitHub repository](https://github.com/monarch-initiative/source-data-analysis).
